draft-regext-rdap-extensions.md

```diff
@@ -386,125 +484,187 @@ that example could be written as:
+
+## Referrals {#referrals}
+
+It is common in the RDAP ecosystem to link from one RDAP resource to
+another, such as can be found in domain registrations in gTLD DNRs.
+These are typically conveyed in the link structure defined in
+[@!RFC9083, section 4.2] and use the "application/rdap+json" media
+type.  For example:
 
     {
-      "rdapConformance": [
-        "rdap_level_0",
-        "lunarNIC"  
-      ],
-      "objectClassName": "domain",
-      "ldhName": "example.com",
-      "links": [
-        {
-          "value": "https://example.com/domain/example.com",
-          "href": "https://example.com/sideways_href",
-          "rel": "sideways",
-          "type": "application/rdap+json"
-        }
-      ]
+      "value": "https://regy.example/domain/foo.example",
+      "rel": "related",
+      "href": "https://regr.example/domain/foo.example",
+      "type": "application/rdap+json"
     }
 
-When defining the usage link relations, extensions should specify the
-media types expected to be used with those link relations.
+Extensions MUST explicitly define any required behavioral changes to
+the processing of referrals.  If an extension does not make any
+provision in this respect, clients MUST assume the information
+provided by referrals requires no additional processing or
+modification to use in the dereferencing of the referral.
 
-Regardless of the category of these extensions, their usage may also
-leverage the appearance of their identifiers in the `rdapConformance` array.
-Clients may use the `/help` query as defined in [@!RFC9082] to discover
-the extensions available. 
+## Extension Versioning {#versioning}
 
-# Extension Versioning
-
-As stated in (#extension_identifier), RDAP extensions are opaque, and
+As stated in (#purpose), RDAP extensions are opaque, and
 they possess no explicit version despite the fact that some extension
 identifiers include trailing numbers. That is, RDAP extensions without
-an explicitly defined versioning scheme are opaquely versioned.
+an explicitly-defined versioning scheme are opaquely versioned.
 
-For example, `fizzbuzz_1` may be the successor to `fizzbuzz_0`, but it
-may also be an extension for a completely separate purpose. Only consultation
-of the definition of `fizzbuzz_1` will determine its relationship with
-`fizzbuzz_0`. Additionally, `fizzbuzz_99` may be the predecessor of `fizzbuzz_0`.
+For example, "fizzbuzz_1" may be the successor to "fizzbuzz_0", but it
+may also be an extension for a completely separate purpose. Only
+consultation of the definition of "fizzbuzz_1" will determine its
+relationship with "fizzbuzz_0". Additionally, "fizzbuzz_99" may be the
+predecessor of "fizzbuzz_0".
 
 If a future RFC defines a versioning scheme (such as using the
-mechanism defined in (#extension_identifier)), an RDAP extension
-definition MUST explicitly denote this compliance.
+mechanism defined in (#purpose)), an RDAP extension definition MUST
```

Comment:
Did we mean the Syntax section here? Further, if we end up scratching the "__" reservation, "(such as using the mechanism defined in ...) can also be scratched.

Reply:
Thanks, yep, updated the reference to 'syntax'.

Reply:
Since "__" has been dropped, the "(such ...)" part has been dropped now as well.